packages/sanity/src/core/studio/packageVersionStatus/PackageVersionStatusProvider.tsx

```diff
@@ -0,0 +1,94 @@
+import {Box, Button, Stack, useToast} from '@sanity/ui'
+import {type ReactNode, useCallback, useEffect, useRef} from 'react'
+import {SANITY_VERSION} from 'sanity'
+import semver from 'semver'
+
+import {hasSanityPackageInImportMap} from '../../environment/hasSanityPackageInImportMap'
+import {useTranslation} from '../../i18n'
+import {checkForLatestVersions} from './checkForLatestVersions'
+
+// How often to to check last timestamp. at 30 min, should fetch new version
+const REFRESH_INTERVAL = 1000 * 60 * 5 // every 5 minutes
```

Comment:
It's quite arbitrary at the moment... since the timeliness of this particular toast isn't so vital, I wasn't so concerned with a bit of drift. We can up the interval!